tests/smoketest/smoketest_collectd_entrypoint.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/sh
-set -e
```

Comment:
Ah yes, I noticed the early exit sometimes today. I usually like `-e` for test scripts like this, but at some point I guess it was no longer correct and we're just noticing now. :)

Reply:
Yea this is kind of an embedded meta-script and since it has control logic for evaluating an exit code of 0 or 1, I figured we should just let them fall through.